server/routes/__tests__/PairingApi.test.js

```diff
@@ -1,40 +1,78 @@
 const request = require('supertest');
 const app = require('../../../server/app');
 
+const PAIRING_ENDPOINT = '/api/pairing';
 describe('Test /api/pairing', () => {
-  test('Sucessful queries return a 200', () => {
+  test('Should return 200 if valid budget param is passed', () => {
     return request(app)
-      .get('/api/')
+      .get('/api/pairing')
```

Comment:
Since you already created the constant ```PAIRING_ENDPOINT```, shouldn't you change this  line to use it to keep it consistent?

Reply:
@AlChiu yep, I fixed it when i restructured the order of the file